<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameSzukaj">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>171</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="poleSzukaj">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="szukajButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Szukaj!</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioTytul">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>tytule</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioAutor">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>autorze</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioRok">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>roku wydania</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioGatunek">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>gatunku</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupWyszukiwanie">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>29</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Wyszukuj po: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="wyswietlWszystko">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyświetl wszystko</string>
     </property>
    </widget>
    <zorder>groupWyszukiwanie</zorder>
    <zorder>poleSzukaj</zorder>
    <zorder>szukajButton</zorder>
    <zorder>radioTytul</zorder>
    <zorder>radioAutor</zorder>
    <zorder>radioRok</zorder>
    <zorder>radioGatunek</zorder>
    <zorder>wyswietlWszystko</zorder>
   </widget>
   <widget class="QFrame" name="frameLista">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>521</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="ksiazkiTabela">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="komunikat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKsiazki">
    <property name="title">
     <string>Książki</string>
    </property>
   </widget>
   <addaction name="menuKsiazki"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
